src/sonic-config-engine/tests/sample_output/py3/qos-arista7800r3-48cq2-lc.json

```diff
@@ -114,6 +114,9 @@
         }
     },
     "PORT_QOS_MAP": {
+        "global": {
+            "dscp_to_tc_map"  : "AZURE"
+        },
```

Comment:
All values in this file need to be updated to use the reference format like "[DSCP_TO_TC_MAP|AZURE]"